buildSrc/build.gradle.kts

```diff
@@ -10,6 +10,15 @@ plugins {
   `kotlin-dsl`
 }
 
+afterEvaluate {
```

Comment:
Hmmm (why?) is this needed? 🤔

Reply:
https://handstandsam.com/2022/04/13/using-the-kotlin-dsl-gradle-plugin-forces-kotlin-1-4-compatibility/

Reply:
Ok pls include this comment in code